types/aphrodite.d.ts

```diff
@@ -27,6 +27,7 @@ declare module "aphrodite" {
          * Pseudo-selectors
          */
         "::placeholder"?: _CSSProperties;
+        ":active"?: _CSSProperties;
```

Comment:
Ohh how didn't we never used `active` in WB! I'm glad that we are now switching gears to pseudo-classes.